common/src/main/java/com/wynntils/models/items/annotators/game/DungeonKeyAnnotator.java

```diff
@@ -19,30 +18,44 @@ public final class DungeonKeyAnnotator implements ItemAnnotator {
     private static final Pattern DUNGEON_KEY_PATTERN =
             Pattern.compile("^(?:§[46])*(?:Broken )?(?:Corrupted )?(.+) Key$");
 
-    private static final Pattern LORE_PATTERN = Pattern.compile("§7(Grants access to the|Use this item at the)");
+    private static final String[] DUNGEONS = {
```

Comment:
I don't like to have a hard-coded list like this. It will break when Wynn adds or renames a dungeon.

Is this since trade market items not always have a lore? In what circumstances might items on the trade market miss the lore? I think there are other annotators as well that will break, since they rely on the lore.

Reply:
Yes, items that fit the second pattern (items in the "View Your Trades" menu) have their lore completely replaced by the trade information. Therefore, annotators cannot rely on lore to match items without some fallback option to match this type of item. I already fixed the other annotators with this problem, but dungeon keys are trickier because the pattern fits any item that ends in the word "key." As far as I know, there is no way to determine if a key item is a dungeon key or not given just the item name without checking each possible name, so this list is an unfortunate necessity.

Reply:
Ok. Like my comment above, this knowledge should really reside in a DungeonModel, but we do not have such either.

Are you interested in trying to add two new models? If so, you can go ahead and do that (ask if you need assistance on what is needed). Otherwise I think we can accept this PR as it is now, and I can create the new models and refactor it later.

Reply:
Sure, I can add more models.